packages/node/src/types/evaluation.ts

```diff
@@ -0,0 +1,11 @@
+export type EvaluationVariant = {
+  key: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  payload?: any;
+};
+
+export type FlagResult = {
```

Comment:
are we using FlagResult for future use cases similar to stuff we have on skylab-api for now? do we need the other things from that class in the nova repo? for now, we can just have the stuff return EvaluationVariant?

Reply:
Yeah the actual return object has more stuff in it. But as of now the only pertinent information for a customer is the variant. For reference: https://github.com/amplitude/experiment-evaluation/blob/main/evaluation-core/src/commonMain/kotlin/FlagResult.kt

The actual flag result contains the `variant`, `description`, and `isDefaultVariant`